onebusaway-android/src/main/java/com/joulespersecond/seattlebusbot/PreferencesActivity.java

```diff
@@ -216,9 +216,14 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
 
             // Wait to change the region preference description until the task callback
             //Analytics
-            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
-                    getString(R.string.analytics_action_button_press),
-                    getString(R.string.analytics_label_button_press_experimental) + experimentalServers);
+            if (experimentalServers)
```

Comment:
@cagryInside Could you please add braces`{` to this IF statement?  As a matter of style I prefer to always use them.  Also, you might want to check if you have the "reformat code" checkbox checked in the "Commit Changes" view in Android Studio.  I think the style template should have automatically reformatted this to include braces.